### ISSUE TITLE
Perform router capture/injection on a separate dedicated interface

### ISSUE THREAD

**Issue by awh** [weaveworks/weave#842]
* Make router capture/inject on a differently named interface

Requires changes to script.

Depends on #841.

**Comment by rade** [comment]
This doesn't really depend on #841.

**Comment by tomwilkie** [comment]
This has been superseded by #826
